servers/parser-graphql-wrapper/src/apollo/resolvers.ts

```diff
@@ -167,86 +146,87 @@ export const resolvers: Resolvers = {
   },
   Query: {
     //deprecated
-    getItemByUrl: async (_source, { url }, { repositories }) => {
+    getItemByUrl: async (_source, { url }, { repositories, dataSources }) => {
       // If it's a special short share URL, use alternative resolution path
       const shortCode = extractCodeFromShortUrl(url);
       if (shortCode != null) {
         const givenUrl = await givenUrlFromShareCode(
           shortCode,
           await repositories.sharedUrlsResolver,
         );
-        const item = getItemByUrl(givenUrl);
+        const item = await dataSources.parserAPI.getItemData(givenUrl);
         item['shortUrl'] = url;
         return item;
       } else {
         // Regular URL resolution
-        return getItemByUrl(url);
+        return dataSources.parserAPI.getItemData(url);
       }
     },
     //deprecated
     getItemByItemId: async (_source, { id }, { repositories }) => {
-      return getItemById(id, await repositories.itemResolver);
+      return null; //getItemById(id, await repositories.itemResolver);
```

Comment:
have the requests stopped for this in studio?

Reply:
yea they have. gonna remove it